pom.xml

```diff
@@ -21,7 +21,7 @@ limitations under the License.
     <groupId>com.microsoft.azure</groupId>
     <artifactId>azure-cosmosdb-spark_2.4.0_2.11</artifactId>
     <packaging>jar</packaging>
-    <version>3.3.4</version>
+    <version>3.3.5-SNAPSHOT</version>
```

Comment:
3.4.0 I would say - new feature. But I can change it when I cut the release tomorrow morning

Reply:
moved to 3.4.0.

Reply:
done.